### ISSUE TITLE
1.10-rc2 task opening, descripion missing?

### ISSUE THREAD

**Issue by markus4000** [osTicket/osTicket#3093]
after adding a task, the text I entered in the description field is missing, and there's no entry in the ost_thread_entry table. Is that a bug or a missconfiguration on my side?

---
Version:
I had 1.9.8, and i upgraded to 1.10rc2 => i had the Problem
then i upgraded to develop-next => Issue is still here.

**Comment by markus4000** [comment]
Does someone else have the same issue?

**Comment by rbukovansky** [comment]
Doesn't work for me either.

**Comment by akashkumardce** [comment]
https://github.com/osTicket/osTicket/pull/3100/commits/e68ec9c01076b4e2e5ee58088627e70946d5b192

Fixes the issue of task page missing description.

**Comment by rbukovansky** [comment]
@akashkumardce  I have patched my 1.10-RC2+git/develop with your changes, but doesn't work for me... It seems to me more like the issue is it doesn't save that text entered in description field. Your patch is more about date and asignee of task.

**Comment by akashkumardce** [comment]
@rbukovansky - Try to create a new task and it might work, Issue is related to duedate. During creation of task duedate entered in db is 0000-00-00 00:00:00 so causing the issue, and the fix is related to that.
If issue is something else, Can you attach a screenshot of the page?

**Comment by rbukovansky** [comment]
I see... But it still doesn't work... See attached screenshots (first one is entering the new task, second one is the result)...

![new_task](https://cloud.githubusercontent.com/assets/1004491/15105254/7faba800-15c0-11e6-9437-ac55b419d1ce.png)
![new_task_result](https://cloud.githubusercontent.com/assets/1004491/15105258/8408e200-15c0-11e6-9d29-99dc258f9020.png)

**Comment by rbukovansky** [comment]
I'm running 1.10-RC2+git/devel

**Comment by t-oster** [comment]
I can confirm this issue. I am running current tip of develop (784184b6dd5846ad9a73c632592dfaaad4b69030).

**Comment by t-oster** [comment]
@akashkumardce your patch does not fix this issue for me.

**Comment by t-oster** [comment]
it definitely works on 9c6589ea529e5240f5845406308c051a21a1202c (develop-next)

**Comment by rbukovansky** [comment]
There's is something really wrong... I have installed 1.10-RC2 from osTicket website from scratch and the tasks are created fine. Then I copied over latest develop-next and creating tasks is broken (task created in distro 1.10-RC2 is shown OK, creating new one is not OK). I can see that row in ost_task for task created in develop-next is created, but the thread entry is not stored in ost_thread_entry.

**Comment by rbukovansky** [comment]
@t-oster Thomas, could you please provide me the .zip file with your osTicket installation? Thank you very much.

**Comment by t-oster** [comment]
even better, here is my current version (including all my mods) at github https://github.com/t-oster/osTicket-1.8/tree/tmp

**Comment by t-oster** [comment]
Ha... thanks to `git bisect` the first **bad** commit is `9616a61d98c4f4fbcdec4b24402a88a936c76c06` aka 
```
commit 9616a61d98c4f4fbcdec4b24402a88a936c76c06
Author: Jared Hancock <jared@osticket.com>
Date:   Tue Dec 29 21:29:46 2015 -0600

    php: Support PHP 7, require at least 5.4
```

**Comment by t-oster** [comment]
Seems to be solved in #3155

**Comment by rbukovansky** [comment]
So, if I understand that correctly, update that just one line and I'm good to go?

**Comment by rbukovansky** [comment]
Yes! Thanks a lot, @t-oster!

**Comment by markus4000** [comment]
Thanks. it works for me! Its still not in RC3... so I let the issue open.

**Comment by markus4000** [comment]
fixed in actual "1.10develop"

**Comment by t-oster** [comment]
yep, this commit fixed it 865db9b58ce63a28e84d673b5af1a260a6603436
